### PR TITLE
Deploy contracts on mainnet

### PR DESCRIPTION
**Pull request by ggobugi27**
- Create `kswap` namespace
- Deploy `exchange.pact`
- Deploy `tokens.pact`

### CODE REVIEW

File: pact/yaml/mainnet/ns.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+codeFile: ../../ns.pact
+signers:
+  - public: 07b7f96e27d5c202c56a56a3e0dea569f2fa9c76274a1c9a19ecc970470011dc
+  - public: 1d062c51850de34ac86e47a70bcbdb68441b530074f8abb51978f6ac1717d5ef
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:4]: stu

File: pact/yaml/mainnet/ns.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - public: 56ff9e5c1b5ec3e98172a76aa3cdc03d3ec664f147267fd99eee4dd09763c853 #doug
+  - public: 7d08eec6c9e52dd39ec8cd29d71b2742aba033482d4a8da2aeeb5e14928f18d6
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:6]: wjm

File: pact/yaml/mainnet/ns.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+codeFile: ../../ns.pact
+signers:
+  - public: 07b7f96e27d5c202c56a56a3e0dea569f2fa9c76274a1c9a19ecc970470011dc
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:3]: wjm

File: pact/yaml/mainnet/ns.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  - public: 1d062c51850de34ac86e47a70bcbdb68441b530074f8abb51978f6ac1717d5ef
+  - public: 56ff9e5c1b5ec3e98172a76aa3cdc03d3ec664f147267fd99eee4dd09763c853 #doug
+  - public: 7d08eec6c9e52dd39ec8cd29d71b2742aba033482d4a8da2aeeb5e14928f18d6
+  - public: a241430816f0112add2179e3e3939ebab5336ab024be990960e684e38eb87d80 #stu
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:7]: use this

File: pact/yaml/mainnet/ns.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+codeFile: ../../ns.pact
+signers:
+  - public: 07b7f96e27d5c202c56a56a3e0dea569f2fa9c76274a1c9a19ecc970470011dc
+  - public: 1d062c51850de34ac86e47a70bcbdb68441b530074f8abb51978f6ac1717d5ef
+  - public: 56ff9e5c1b5ec3e98172a76aa3cdc03d3ec664f147267fd99eee4dd09763c853 #doug
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:5]: use this

File: pact/yaml/mainnet/ns.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    keys:
+      - 8cc8da8ea7cbfaf6510dbc5f402b025dceaa9a3eaf7145e0ba933d468b63d358 #heekyun
+      - a241430816f0112add2179e3e3939ebab5336ab024be990960e684e38eb87d80 #stu
+      - 4b0f29b9e0a996587e5b5524731c91ecf02ecaa8b7e70e5f8f1881c5f0c18fc1 #francesco
+    pred: keys-any
+networkId: mainnet01
+nonce: install-kswap-namespace
+publicMeta:
+  chainId: "2"
+  sender: # add sender
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:26]: no opinion. I have a gas acct 9a4849687cbcfeb1f7c6510539638da576289508aedcc75f4d6ad3ed2623635c

File: pact/yaml/mainnet/ns.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  gasLimit: 600
```
Comment (sirlensalot) [inline pact/yaml/mainnet/ns.yaml:27]: This probably needs to be bigger :)

File: pact/yaml/mainnet/ns-unsigned.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+hash: jndqRL-b-7-Zb3P9oO3-aHxiXVg1YqvOXmt-9WTN7vk
```
[the remark this answers is not in the document]
Comment (ggobugi27) [reply pact/yaml/mainnet/ns-unsigned.yaml]: Removed

File: pact/yaml/mainnet/ns.yaml (hunk 7 above)
Comment (ggobugi27) [reply pact/yaml/mainnet/ns.yaml]: I saw that we used 600 for creating `util` namespace, but changed to 1200.

File: pact/yaml/mainnet/ns.yaml (hunk 6 above)
Comment (ggobugi27) [reply pact/yaml/mainnet/ns.yaml]: added in my acct `8cc8da8ea7cbfaf6510dbc5f402b025dceaa9a3eaf7145e0ba933d468b63d358`

**Review by sirlensalot — APPROVED** [review]

**Pull request merged**